src/System.IO.FileSystem/tests/Directory/CreateDirectory.cs

```diff
@@ -33,16 +33,24 @@ public void EmptyAsPath_ThrowsArgumentException()
             Assert.Throws<ArgumentException>(() => Create(string.Empty));
         }
 
-        [ActiveIssue(25665)]
         [Theory, MemberData(nameof(PathsWithInvalidCharacters))]
         public void PathWithInvalidCharactersAsPath_ThrowsArgumentException(string invalidPath)
```

Comment:
Seems like the test name is now wrong, too.  It's not necessarily throwing ArgumentException.